packages/graph/hash_graph/bin/hash_graph/Cargo.toml

```diff
@@ -8,16 +8,15 @@ description = "The entity-graph query-layer for the HASH datastore"
 
 
 [dependencies]
-axum = "0.5.11"
-clap = { version = "3.2.10", features = ["cargo", "derive", "env", "wrap_help"] }
-clap_complete = "3.2.3"
-# TODO: Change to `version = "0.2"` as soon as it's released
```

Comment:
We still haven't swapped to 0.2 in the engine and such, didn't want to do it in this pr. cc @TimDiekmann